tests/components/omnilogic/const.py

```diff
@@ -0,0 +1,266 @@
+"""Constants for the Omnilogic integration tests."""
+
+TELEMETRY = {
```

Comment:
If a code-owner can get a proper fixture file (ie. the result from `get_telemetry_data`) then it would be better.
But for now I used this DEBUG object from an issue report.

Reply:
I am no longer involved in managing the Omnilogic integration that is built into Home Assistant. I have requested that I be removed or the integration be removed as I will not be contributing to it going forward. We attempted to continue to build out this integration to support other services, but the turnaround was too slow given the low use of this integration - which is fine and I completely understand it.

Reply:
If you do not wish to be a code-owner anymore, then you should open a simple PR to remove yourself from the manifest file.

Reply:
Thanks @epenet - done.